ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml

```diff
@@ -0,0 +1,35 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
```

Comment:
I don't think we want this step to depend on CVP images. What image would typically contain operator-sdk binary?